omnisafe/algorithms/on_policy/policy_gradient.py

```diff
@@ -0,0 +1,466 @@
+# Copyright 2022 OmniSafe Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implementation of the PolicyGradient algorithm."""
```

Comment:
Use formal algorithm name in the docstring, not the class name.

Reply:
Already modify the same problem with another algorithm, e.g. convert `Implementation of the TRPO-Lag algorithm.` to `Implementation of the lagrange version of TRPO algorithm.`.